Doogee/V30T/res/xml/power_profile.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="ambient.on">0.1</item>
+    <item name="screen.on">0.1</item>
+    <item name="screen.full">0.1</item>
```

Comment:
that's obviously wrong

Reply:
Hi, thanks for reply. I retrieved this xml file from vendor of official rom.